Gemfile

```
@@ -55,6 +55,8 @@ group :test do
   gem 'faker'
   # For clearing db before each test run
   gem 'database_cleaner'
+  # Add SimpleCov for code coverage
+  gem 'simplecov', require: false, group: :test
```

You need to lock this to v0.17 for now:

```suggestion
  gem 'simplecov', '~> 0.17.1'
```

As a general note I always like to lock gems down so they cannot automatically update their major version without us making a change.